src/scene/batching/skin-batch-instance.js

```diff
@@ -46,8 +45,8 @@ Object.assign(SkinBatchInstance.prototype, {
             mp[base + 11] = pe[14];
         }
 
-        SkinInstance.prototype.uploadBones.call(this, this.device);
+        super.uploadBones(this.device);
```

Comment:
Can't this just be `this.uploadBones(this.device);`, with the behavior being to look up the class hierarchy to the superclass automatically?

Reply:
It should be `super`, otherwise it will start lookup from `this`, which will point to same methods.

Reply:
But it won't find `uploadBones` on this, so will then look for the function on the superclass.

Reply:
Should do, you are right. Weird it used `prototype...call` before, as being inherited, it should've worked with `this.uploadBones` as well.

Reply:
changed to this. uploadBones. No need for super.init and this does not implement uploadBones.